scripts/vsce-bundled-extension.ts

```diff
@@ -0,0 +1,118 @@
+import {
```

Comment:
THis is the script that does the vsce bundling now. Run via npm script.  See execution in action on the build-all circleci build